docs/reference/project-configuration.md

```diff
@@ -11,11 +11,11 @@ The file must be called `spatialos.json` and must have the following structure:
   "projectName": "demo",
   "schemaDescriptor": "./schema/bin/schema.descriptor",
   "clientWorkers": [
-    "../OtherWorkers/Interactive/client/improbable/client_worker.json"
+    "../client/improbable/client_worker.json"
```

Comment:
(The worker jsons are not located in the SpatialOS folder yet. (See the design doc)

Reply:
good point! changing..

Reply:
done